students/axel_nowaczyk/lesson_01_basics/school_desks.py

```diff
@@ -0,0 +1,20 @@
+from functools import reduce
+
+
+def school_desks():
+    a = int(input())
+    b = int(input())
+    c = int(input())
+    sum = reduce((lambda x, y: x + number_of_desks(y)), [0, a, b, c])
```

Comment:
Change sum variable name because sum is built in function:
>>> sum([1,2,3])
6
>>> sum = 10
>>> sum([1,2,3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
>>>